index.html

```diff
@@ -685,6 +685,10 @@ <h2 class="title-section">Community Partners</h2>
 
     <div class="sponsors-list-wrapper">
       <div class="sponsors-list" id="sponsors-carousel" style="text-align: center">
+        <!-- item -->
+        <div class="sponsor-item platinum">
+          <a href="https://asetalias.in/"><img src="img/alias.png" alt="ALiAS" class="sponsor-img" style="max-width: 100%;object-fit: contain;"></a>
```

Comment:
Inline `style` property not required. I checked it, it looks fine on both desktop and mobile.
```suggestion
          <a href="https://asetalias.in/"><img src="img/alias.png" alt="ALiAS" class="sponsor-img"></a>
```
- Mobile:
![Screenshot 2023-08-06 at 15-03-42 PyDelhi Conference](https://github.com/pydelhi/conference/assets/34956994/54ae0d16-d4fc-457c-aa28-7cafe2285811)

- Desktop:
![image](https://github.com/pydelhi/conference/assets/34956994/3cb86def-499f-451d-820a-9334fd4578c6)